content/docs/tutorials/azure/azure-ts-appservice-devops.md

```diff
@@ -2,7 +2,7 @@
 title: "A Todo App on Azure App Service with SQL Database and Application Insights and deploys it to Azure DevOps"
 ---
 
-<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-devops" target="_blank">
+<a href="https://app.pulumi.com/new?template=https://github.com/pulumi/examples/tree/master/azure-ts-appservice-devops/infra" target="_blank">
```

Comment:
This example doesn't have the button on it's README.md, so I lean towards wishing there were a way to opt-out of showing it here, but this appears to work, so I suppose we could keep it. But we should consider updating the README.md to include it as well.

Reply:
I will add it.

Reply:
I created a new PR to add it to the README: https://github.com/pulumi/examples/pull/380